src/lib/utils/assert.cpp

```diff
@@ -6,7 +6,13 @@
 */
 
 #include <botan/exceptn.h>
+#include <botan/build.h>
 #include <sstream>
+#include <cstdlib>
```

Comment:
Is the `cstdlib` required in the general case or only with `BOTAN_TERMINATE_ON_ASSERTS`?